tools/JsonCli/.azure-pipelines/main.yml

```diff
@@ -3,8 +3,19 @@ jobs:
   pool:
     name: VSEng-MicroBuildVS2019
   variables:
-    ProjectPath: 'tools/JsonCli/src/Microsoft.TemplateEngine.JsonCli.csproj'
+    SrcPath: 'tools/JsonCli/src'
+    ProjectPath: '$(SrcPath)/Microsoft.TemplateEngine.JsonCli.csproj'
+    SigningProjectPath: '$(SrcPath)/Signing.csproj'
+    NugetConfigPath: '$(SrcPath)/nuget.config'
+    DropPath: '$(build.artifactstagingdirectory)/drop'
   steps:
+  - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@2
```

Comment:
What's the significance of this task name?

Reply:
I have no idea lol. When I'm working on this, I use the editor that exists within Azure Dev Ops to figure out what the YAML should be. I search for the task, add it, and use the UI to change some of the settings. It looks something like this:
![Screen Shot 2020-10-01 at 12 31 00 PM](https://user-images.githubusercontent.com/11282622/94854651-197c4980-03e2-11eb-9107-e6834c2b6c00.png)

Once I'm done, I click the "View YAML" button which gives me the task name and other formatting stuff:
![Screen Shot 2020-10-01 at 12 31 06 PM](https://user-images.githubusercontent.com/11282622/94854716-30bb3700-03e2-11eb-8f53-52cd98b055e7.png)